### DISCUSSION TITLE
电子文献/在线文献的著录格式在网址链接之前如何著录最近访问日期

### DISCUSSION THREAD

**Discussion by sjtuzwj** [sjtug/SJTUThesis#672] [提问] [answered]
举例：江向东. 互联网环境下的信息处理与图书管理系统解决方案[J/OL]. 情报学报,1999,18(2);4.[2000-01-18].http://www.chinainfo.gov.cn/periodical/gbxb. 

...好迷啊

**Comment by sjtuzwj** [comment]
图书馆专家审核论文定稿给出的需求.另外DOI也不让打

**Reply by sjtuzwj** [reply]
如果没有解决方案的话我就直接加URL里了,链接失效就失效吧..

**Comment by skyzh** [comment, marked as answer]
SJTUThesis 文献管理使用 biblatex-gb7714-2015 包。参考 https://mirrors.sjtug.sjtu.edu.cn/ctan/macros/latex/contrib/biblatex-contrib/biblatex-gb7714-2015/biblatex-gb7714-2015.pdf

> 此外用 date 表示更新或修改日期，urldate 表示引用 (访问) 日期。如果出现复杂情况，更新或修改日期还可以利用 enddate/eventdate 表示。注意 修改日期需要表示到日

**Reply by skyzh** [reply]
现在的 thesis.bib 里面也有

```bibtex
@online{xiaoyu2001,
  author  = {萧钰},
  url     = {http://www.creader.com/news/20011219/200112190019.html},
  date    = {2001-12-19},
  title   = {出版业信息化迈人快车道},
  urldate = {2002-04-15},
}
```
